src/Util/Stokes.h

```diff
@@ -38,28 +95,75 @@ class Stokes {
 // We usually don't want to calculate the whole image data, because it spends a lot of time.
 // StokesSource will bind casacore::Slicer or casacore::ImageRegion, because the coordinate of a computed stokes image is different from
 // the original image coordinate.
+
+/**
+ * @brief Represents a source of Stokes data with axis range specifications.
+ *
+ * This struct encapsulates information about a Stokes parameter and its associated
+ * axis ranges in an image dataset. It is used to tell the file loader to get the original
+ * image interface, or get the computed stokes image interface.
+ * It provides constructors for different levels of
+ * specification and utility functions for comparison and checking if the data
+ * represents an original image.
+ */
 struct StokesSource {
-    int stokes;
-    AxisRange z_range;
-    AxisRange x_range;
-    AxisRange y_range;
+    int stokes;        ///< The Stokes parameter identifier.
+    AxisRange z_range; ///< The range along the Z-axis.
```

Comment:
We should standardise on either e.g. "Z-axis" or "z-axis" in all the docs -- I don't really mind which one as long as it's consistent. I have a slight preference for the capitalised version.

Reply:
@confluence if Z-axis then also X-axis and Y-axis?

Reply:
> @confluence if Z-axis then also X-axis and Y-axis?

Yes. Only in these docstrings; it doesn't matter what we do in internal comments.